src/main/java/org/gridsuite/modification/server/service/NetworkStoreListener.java

```diff
@@ -236,8 +246,8 @@ public void onRemoval(Identifiable identifiable) {
         //equipmentInfosService.delete(identifiable.getId(), networkUuid);
     }
 
-    public void setSubstationsIds(Set<String> substationsIds) {
-        this.substationsIds = substationsIds;
+    public void addSubstationsIds(Set<String> substationsIds) {
```

Comment:
Remove it and add VLs in the `onCreation` and the `deleteEquipmentInfos` methods
and rename `deleteEquipmentInfos` by `onTemporaryRemoval`

Reply:
Done in https://github.com/gridsuite/network-modification-server/pull/67